src/Collapse.svelte

```diff
@@ -13,23 +14,44 @@
   export let onEntered = noop;
   export let onExiting = noop;
   export let onExited = noop;
+  export let expand = false;
 
   const props = clean($$props);
 
-  let _wasOpen = isOpen;
   $: classes = clsx(
     className,
     // collapseClass,
     navbar && 'navbar-collapse',
   );
 
   let windowWidth = window.innerWidth;
+  let _wasMaximazed = false;
 
-  $: if (windowWidth >= 768 && navbar && _wasOpen === isOpen) {
-    _wasOpen = isOpen;
-    isOpen = true;
-  } else if (windowWidth < 768) {
-    isOpen = _wasOpen;
+  const minWidth = {};
+  minWidth['xs'] = 0;
+  minWidth['sm'] = 576;
+  minWidth['md'] = 768;
+  minWidth['lg'] = 992;
+  minWidth['xl'] = 1200;
```

Comment:
This is a good improvement over current behavior, though one comment is can you think of a way that we can avoid hardcoding these breakpoints?

The concern is that these are actually sass variables and could be different in some Bootstrap themes:

https://github.com/twbs/bootstrap/blob/master/scss/_variables.scss#L186

Reply:
You might create a div element inside body tag like this to find which viewport used (xs, sm, md, lg, or xl)
```
<div id="sizer">
    <div class="d-block d-sm-none d-md-none d-lg-none d-xl-none" data-size="xs"></div>
    <div class="d-none d-sm-block d-md-none d-lg-none d-xl-none" data-size="sm"></div>
    <div class="d-none d-sm-none d-md-block d-lg-none d-xl-none" data-size="md"></div>
    <div class="d-none d-sm-none d-md-none d-lg-block d-xl-none" data-size="lg"></div>
    <div class="d-none d-sm-none d-md-none d-lg-none d-xl-block" data-size="xl"></div>
</div>
```

and then
```
const breakpoint = document.querySelector("#sizer").querySelectorAll("div:visible").data("size")
```

Reply:
Thanks for the idea 👍 will take a look